client/src/components/Draftail/CommentableEditor/CommentableEditor.js

```diff
@@ -0,0 +1,217 @@
+import { DraftailEditor, ToolbarButton, createEditorStateFromRaw, serialiseEditorStateToRaw } from 'draftail';
+import { EditorState, Modifier, RichUtils, SelectionState } from 'draft-js';
+import { filterInlineStyles } from "draftjs-filters";
```

Comment:
This should be added as a direct dependency in `package.json` if you use it here. Draftail won’t stop depending this anytime soon, but it’s more correct to declare all dependencies.

Reply:
Good point, thanks! Will address this